app/react/sidebar/SidebarItem/SidebarTooltip.tsx

```diff
@@ -0,0 +1,24 @@
+import Tippy, { TippyProps } from '@tippyjs/react';
+
+type Props = {
+  content: React.ReactNode;
+  children?: TippyProps['children'];
+};
+
+export function SidebarTooltip({ children, content }: Props) {
```

Comment:
we can't use `ToolTipWithChildren` or other tooltip component?

Reply:
I gave it a shot and I'd have to make the following changes:

- add delay and duration override props to get desired behaviour
- remove a lot of the !important styles that are given to tooltip with children so that the tailwind classnames that I pass as props, get applied

I'd prefer to keep as a separate component to not remove the !important styles to avoid introducing bugs with the main tooltipwithchildren component.